style.css

```diff
@@ -117,8 +123,298 @@ body {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+  margin-left: 20px;
+}
+
+.description ul {
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.languages > li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  color: #fff;
+  margin: 0;
+}
+
+.frameworks > li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  color: #fff;
+  margin: 0;
+}
+
+.skills > li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 30px;
+  color: #fff;
+  margin: 0;
+}
+
+.description ul > li {
+  list-style-type: none;
+  margin-left: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  color: #1a2236;
+  text-align: center;
+  line-height: 2;
+}
+
+.languages ul > li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  color: #fff;
+}
+
+.frameworks ul > li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  color: #fff;
+}
+
+.skills ul > li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 27px;
+  letter-spacing: -0.0525em;
+  color: #fff;
+}
+
+.languages li:nth-child(1) {
+  list-style-type: none;
+  padding-bottom: 21px;
+}
+
+.frameworks li:nth-child(1) {
+  list-style-type: none;
+  padding-bottom: 21px;
+}
+
+.skills li:nth-child(1) {
+  list-style-type: none;
+  padding-bottom: 21px;
+}
+
+.description ul li:nth-child(1) {
+  width: 69px;
+  height: 20px;
+  background: #98dede;
+}
+
+.description ul li:nth-child(2) {
+  width: 87px;
+  height: 20px;
+  background: #98dede;
+}
+
+.description ul li:nth-child(3) {
+  width: 74px;
+  height: 20px;
+  background: #98dede;
+}
+
+.description button {
+  width: 178px;
+  height: 44px;
+  background-color: #1a2236;
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: center;
+  margin-left: 20px;
+  border-color: #fff;
+}
+
+.about button {
+  width: 288px;
+  height: 50px;
+  background-color: #1a2236;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  margin-top: 20px;
+  border-color: #fff;
+}
+
+.description button:hover {
+  background-color: #fff;
+  color: #000;
+}
+
+.description button img {
+  width: 18px;
+  height: 11.35px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-left: 16px;
+}
```

Comment:
- [ ] [OPTIONAL] Great work implementing all the changes suggested by previous reviewers 💯. However, I noticed the About me section doesn't have the correct right margin 🤓. Kindly consider applying the correct margin to stick as much as possible to the Figma design. 🔧
    > Hint : try replacing your css property `margin-left: 16px;` for `margin: 0 16px` in your `.about` class.

This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/235313093-a5af94be-5085-4052-8df8-e9819a581652.png)

This is your design 👇🏽 (almost there 😅)

![design](https://user-images.githubusercontent.com/116701485/235313088-22fad783-bbc7-41ef-b07d-dc4deec4c0cb.png)